tests/data/pia.asc

```diff
@@ -0,0 +1,38 @@
+;	V3 text file written for MicroBrightField products.
+(ImageCoords)
+
+("pia"
+  (Closed)
+  (MBFObjectType 5)
+  (0 1 2 3)
+  (3 4 5 4)
+  (6 7 8 5)
+  (9 10 11 6)
+ )
+
+("layer1-2"
+  (Color RGB (255, 251, 255))
+  (GUID "11594A5D0D7646B580E877CB40957E07")
+  (MBFObjectType 5)
+  (Resolution 1.846036)
```

Comment:
Sorry, I don't understand your proposal.

Reply:
I mean those fields:
```
  (GUID "11594A5D0D7646B580E877CB40957E07")
  (MBFObjectType 5)
  (Resolution 1.846036)
```
Why are they here? Should they be loaded as well? I see them for the first time in morphologies.

Reply:
Me too. It's from a file Lida gave me. Maybe one day they will be parsed but for the time being, there is no need.

Reply:
It could be some addition in latest NeuroLucida releases.

Reply:
ok then

Reply:
There is a variety of info in the raw files that we get from reconstructors, so this is just a subset of the info saved in the file. However, at the moment we don't need to load the rest of them because they differ between version, reconstructor and lab so it will be hard to capture all possible combinations.